### PR TITLE
Palindrome.md: Add a step to clean the string

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by cclauss** [comment]
What is the reason that we assign PRs back to the PR author? It seems like a busy work step to me.  We already know that the same user is the author of the PR.

**Comment by UG-SEP** [comment]
> What is the reason that we assign PRs back to the PR author? It seems like a busy work step to me. We already know that the same user is the author of the PR.

I think @cclauss 
it is to show that we are a active, discipline and follow the rules

**Comment by UG-SEP** [comment]
@cclauss  there is one more way to check palindrome with space complexity O(1)

**Review by UG-SEP — APPROVED** [review]
LGTM 👍🏼

**Review by UG-SEP — APPROVED** [review]

**Comment by cclauss** [comment]
I am not questioning the activity or the discipline.  I am questioning why the rule is useful.

**Review by aayushjain7 — APPROVED** [review]

File: docs/en/Strings/Palindrome.md [hunk 1]
```diff
@@ -28,8 +29,9 @@ Compare the reversed string to the original string, if they are equal this mean
 - [Python](../../../algorithms/Python/strings/palindrome.py)
 - [Rust](../../../algorithms/Rust/strings/palindrome/src/main.rs)
 
-## Video URL
-[A coursera video explaining the palindrome Algorithm](https://www.coursera.org/lecture/program-code/palindrome-algorithm-1-zzQqs) 
+## URLs
+* https://en.wikipedia.org/wiki/Palindrome
```
Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:33]: Could move this to others, and make as markdown link?

File: docs/en/Strings/Palindrome.md [hunk 2]
```diff
@@ -28,8 +29,9 @@ Compare the reversed string to the original string, if they are equal this mean
 - [Python](../../../algorithms/Python/strings/palindrome.py)
 - [Rust](../../../algorithms/Rust/strings/palindrome/src/main.rs)
 
-## Video URL
-[A coursera video explaining the palindrome Algorithm](https://www.coursera.org/lecture/program-code/palindrome-algorithm-1-zzQqs) 
+## URLs
```
Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:32]: This is a video URL, do not change the section name

**Review by ming-tsai — CHANGES_REQUESTED** [review]

**Comment by ming-tsai** [comment]
Hi @cclauss,
I have had add some comments.

File: docs/en/Strings/Palindrome.md (hunk 2 above)
Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:32]:
```suggestion
## Video URL
```

File: docs/en/Strings/Palindrome.md (hunk 1 above)
Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:33]:
```suggestion

```

File: docs/en/Strings/Palindrome.md [hunk 3]
```diff
@@ -28,8 +29,9 @@ Compare the reversed string to the original string, if they are equal this mean
...
+* https://en.wikipedia.org/wiki/Palindrome
+* [A coursera video explaining the palindrome Algorithm](https://www.coursera.org/lecture/program-code/palindrome-algorithm-1-zzQqs) 
 
 ## Others
```
Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:37]:
```suggestion
[Wikipedia](https://en.wikipedia.org/wiki/Palindrome)
```

Comment (ming-tsai) [inline docs/en/Strings/Palindrome.md:37]:
```suggestion
[Wikipedia](https://en.wikipedia.org/wiki/Palindrome)
```

**Comment by cclauss** [comment]
Why do we privilege video over other forms of communication?  When I was studying this page, I was on public transportation where video learning was not an option.  I doubt that we should assume that learners can learn fastest/best via video or are in an environment where video is an option.

**Comment by ming-tsai** [comment]
> Why do we privilege video over other forms of communication? When I was studying this page, I was on public transportation where video learning was not an option. I doubt that we should assume that learners can learn fastest/best via video or that they are in an environment where video is an option.

First, watching videos is faster than reading documentation.
Second, the other link is like something more like if you want to learn by your self

**Review by ming-tsai — APPROVED** [review]
Thanks @cclauss

**Pull request merged**